partiql-cli/src/main/kotlin/org/partiql/cli/pico/PipelineOptions.kt

```diff
@@ -70,14 +70,22 @@ internal class PipelineOptions {
     )
     var undefinedVarBehavior: UndefinedVariableBehavior = UndefinedVariableBehavior.ERROR
 
+    @CommandLine.Option(
+        names = ["--plugins"],
+        description = ["Directory containing plugin jar files"],
+        paramLabel = "DIR"
+    )
+    var pluginDir: String = System.getProperty("user.home") + "/.partiql/plugins"
```

Comment:
Look at `PipelineOptions::environmentFile`. It could be useful to use a File instead. Then, when you create the default, you can use `File::resolve(".partiql").resolve("plugins")` -- which is system agnostic.